app/assets/hub/kong-inc/rate-limiting-advanced/fixed-window-1.yaml

```diff
@@ -0,0 +1,9 @@
+description: |
```

Comment:
Can we use a representative filename? It's likely that the filename will form part of the API e.g. `GET /api/plugin-examples/FILENAME`

`fixed-window-500-per-hour` makes more sense than `fixed-window-1` to me